elftools/elf/descriptions.py

```diff
@@ -358,6 +373,7 @@ def describe_attr_tag_arm(tag, val, extra):
     SHT_GNU_verneed='VERNEED',
     SHT_GNU_versym='VERSYM',
     SHT_GNU_LIBLIST='GNU_LIBLIST',
+    # SHT_LLVM_ADDRSIG='LLVM_ADDRSIG',
```

Comment:
Is this needed?

Reply:
I probably wont have time to add LLVM ADDR SIG support; I've added a commit on top removing them.  Feel free to squash when merging; if it's kept separate the values will be in git history at least.